app/src/main/java/com/mobiledevelopmentfoodapp/MainViewModel.kt

```diff
@@ -10,12 +9,12 @@ import kotlinx.coroutines.launch
 
 class MainViewModel : ViewModel() {
 
-    var restaurant : MutableLiveData<List<Food>> = MutableLiveData<List<Food>>()
-    var RestaurantService : RestaurantService = RestaurantService()
```

Comment:
Implemented proper camel case here per variable names best practices. Best practices can be found here. https://kotlinlang.org/docs/coding-conventions.html#property-names

Reply:
Good.